sources/engine/Stride.Rendering/Rendering/Images/Fog/Fog.cs

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Xenko contributors (https://xenko.com) and Silicon Studio Corp. (https://www.siliconstudio.co.jp)
```

Comment:
Should be changed to `// Copyright (c) Stride contributors (https://stride3d.net)`
(no need for Silicon Studio as they didn't participate for this file)